framework_tvb/tvb/interfaces/rest/server/README.md

```diff
@@ -0,0 +1,96 @@
+
+### Create client
+
+3. Now we'll create a client for the TVB REST server inside _myrealm_. Open [Keycloak Admin Console](http://localhost:8081/admin) 
+
+    1. Click `Clients` (left-hand menu)
+    2. Click `Create` (top-right corner of table)
+    3. Click `Import` and select [tvb-rest.json](./keycloak_configs/tvb-rest.json) file and then click on `Save`
+
+### Create User
+
+4. We also need a user inside _myrealm_ so let's create a user. Open [Keycloak Admin Console](http://localhost:8081/admin)
+
+    1. Click `Users` (left-hand menu)
+    2. Click `Add user` (top-right corner of table)
+    3. Provide values for required fields (username, first and last name) and click on `Save`
+    4. Now set the password for this user, click `Credentials` (on top of page)
+    5. Set the password, you can also make it temporary to avoid resetting the password on first login
+    6. Remember the username and password because you'll need this for login when using TVB REST server
+
+### Configure TVB
+
+5. Now we've to configure the TVB to run the REST server on app start. So for that start your main TVB app and navigate to [TVB Settings](http://localhost:8080/settings/settings/)
+
+    1. To use TVB app using TVB user and rest server using keycloak user. Do following
+        1. Enter [keycloak_config.json](./keycloak_configs/keycloak_config.json) in the `Rest API Keycloak configuration file` field. And make sure [keycloak_config.json](./keycloak_configs/keycloak_config.json) is accessible to the TVB app
+        
+        ![This is an image](./keycloak_configs/keycloak_setting_rest_only.jpg)
+
+    2. To use the same keycloak user in the TVB app and in the rest server
+        1. Enable the keycloak login, click on checkbox
+        2. Enter the same [keycloak_config.json](./keycloak_configs/keycloak_config.json) file name in the `Web Keycloak configuration file` field
+                
+
+        ![This is an image](./keycloak_configs/keycloak_setting_rest_and_gui.jpg)
+
+    Click `Apply` to save changes and restart TVB.
+
+
+If you've enabled keycloak login then after restart TVB will open the keycloak login window and you'll have to login with the username and password you've created earlier. 
+
+If TVB rest server started successfully then you'll be able to see the rest server documentation at [http://localhost:9090/doc/](http://localhost:9090/doc)
+
+Test the rest server, by firing a simulation from command line:
+
+```
+python -m tvb.interfaces.rest.client.examples.fire_simulation --rest-url=http://localhost:9090
+```
+ 
+Or using [tvb-rest-client](https://pypi.org/project/tvb-rest-client/)
+
+```
+from tvb.interfaces.rest.client.tvb_client import TVBClient
+tvb_client = TVBClient("http://localhost:9090")
+tvb_client.browser_login()
+print(tvb_client.get_project_list())
+```
+
+
+
```

Comment:
Include a phrase to inform the user that operations launched within the REST server, are visible in the GUI as well.